packages/@aws-cdk/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -431,3 +431,187 @@ describe('test with shared setup', () => {
     });
   });
 });
+
+describe('pipeline with codestar notification integration', () => {
```

Comment:
Can we move this to a separate, new test file, like you did with CodeBuild?

Reply:
Already separated to a new test file.